core/server/worker/src/main/java/alluxio/worker/grpc/BlockReadHandler.java

```diff
@@ -167,7 +172,8 @@ private void openBlock(BlockReadRequestContext context, StreamObserver<ReadRespo
 
         // When the block does not exist in Alluxio but exists in UFS, try to open the UFS block.
         Protocol.OpenUfsBlockOptions openUfsBlockOptions = request.getOpenUfsBlockOptions();
-        if (mWorker.openUfsBlock(request.getSessionId(), request.getId(),
+        if (openUfsBlockOptions != null &&
```

Comment:
Protobuf does not allow null values so this check is unnecessary?

Reply:
done